.github/workflows/manifest-check.yml

```diff
@@ -0,0 +1,36 @@
+name: Manifest check
+
+on:
+  workflow_call:
+    secrets: {}
+    inputs:
+      MANIFEST_IMAGE_TAG:
+        required: true
+        type: string
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  manifest-check:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/jonzudell/manifest/manifest:${{ inputs.MANIFEST_IMAGE_TAG }}
+      options: --user 1001
```

Comment:
Why 1001 here?

Reply:
1001 is the UID for manifestuser in the container
```
# Sets up manifestuser as user 1001 ready to execute manifest
RUN adduser -D -u 1001 manifestuser
USER manifestuser:1001
```

Without this git complains of dubious repo ownership when executing in actions.

Reply:
Hmmm, that's a little obtuse by possibly fine for now.